server/plugin.go

```diff
@@ -162,6 +162,16 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 		return nil, ""
 	}
 
+	shouldProcessMessage, e := p.Helpers.ShouldProcessMessage(post)
```

Comment:
Nit pick: Using `e` as a name for an `error` is quite uncommon. I would suggest using `error` and renaming the `*model.Apperror` in L176 into `appError`.

Reply:
yeah, I thought that would be better too, I just tried not to change any outside code